e2e-disabled/AccountListener.spec.ts

```diff
@@ -22,16 +22,16 @@
  * SOFTWARE.
  */
 
-import {AccountListener} from "../../src/infrastructure/AccountListener";
-import {TransactionHttp} from "../../src/infrastructure/TransactionHttp";
-import {Account} from "../../src/models/account/Account";
-import {Address} from "../../src/models/account/Address";
-import {XEM} from "../../src/models/mosaic/XEM";
-import {NetworkTypes} from "../../src/models/node/NetworkTypes";
-import {EmptyMessage} from "../../src/models/transaction/PlainMessage";
-import {TimeWindow} from "../../src/models/transaction/TimeWindow";
-import {TransferTransaction} from "../../src/models/transaction/TransferTransaction";
-import {NEMLibrary} from "../../src/NEMLibrary";
+import {AccountListener} from "../src/infrastructure/AccountListener";
+import {TransactionHttp} from "../src/infrastructure/TransactionHttp";
+import {Account} from "../src/models/account/Account";
+import {Address} from "../src/models/account/Address";
+import {XEM} from "../src/models/mosaic/XEM";
+import {NetworkTypes} from "../src/models/node/NetworkTypes";
+import {EmptyMessage} from "../src/models/transaction/PlainMessage";
+import {TimeWindow} from "../src/models/transaction/TimeWindow";
+import { TransferTransaction, TxType } from "../src/models/transaction/TransferTransaction";
```

Comment:
Let's probably be more verbose on this type TxType - maybe TransactionType or whatever it is